### PR TITLE
OSOE-548: List sdks

### PR DESCRIPTION
**Pull request by dministro**
[OSOE-548](https://lombiq.atlassian.net/browse/OSOE-548)

[OSOE-548]: https://lombiq.atlassian.net/browse/OSOE-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Piedone — APPROVED** [review]

File: .github/actions/print-config-summary/action.yml
Comment (dministro) [inline .github/actions/print-config-summary/action.yml:1]: @Psichorex as I see these changes are not utilized. Do we need these changes?

Comment (Psichorex) [reply .github/actions/print-config-summary/action.yml:1]: Most possibly we don't need them

Comment (Piedone) [reply .github/actions/print-config-summary/action.yml:1]: These are fine, just more debug output.

Comment (dministro) [reply .github/actions/print-config-summary/action.yml:1]: ~~Then, please close this PR and point to the dev in the super project and in the workflows.~~

I'm just noticed @Piedone answer now. This change is not utilized, so at least, please point to the issue branch in the workflows to test and utilize the action changes.
Eg.: https://github.com/Lombiq/GitHub-Actions/blob/ebc33e763755cc50fd0a22499aa1c61373799b05/.github/workflows/build-and-test-orchard-core.yml#L220-L222

```diff
      - name: Print configuration summary
        if: inputs.print-config-summary == 'true' && (success() || failure())
-         uses: Lombiq/GitHub-Actions/.github/actions/print-config-summary@dev
+         uses: Lombiq/GitHub-Actions/.github/actions/print-config-summary@issue/OSOE-548
```

Comment (Piedone) [reply .github/actions/print-config-summary/action.yml:1]: We'll wait until everything else is done with that :).

Comment (Psichorex) [reply .github/actions/print-config-summary/action.yml:1]: This change was due to trying to find out if there were any SDK problems mentioned by you @dministro in a previous Teams channel conversation. 
The `pring-config-summary` configs had been removed and the target branch was re-aimed at dev.
It up to @Piedone whether we want to keep the `--list-sdks` or not.

Comment (Piedone) [reply .github/actions/print-config-summary/action.yml:1]: As I said, we can keep this. But as Jocó says, indeed change the branch reference so it's actually utilized and its output shows up under the OSOCE PR's builds.

Comment (Piedone) [reply .github/actions/print-config-summary/action.yml:1]: I'll do this.

**Review by Piedone — APPROVED** [review]

**Pull request merged**